### PR TITLE
#166151263 Upgrade Nutrition Diary

### PR DESCRIPTION
**Pull request by wekesa931**
### What does this PR do?
Enables the user to enter the food he/she ate during the day, in addition to what was planned with the nutritional plan.

### Description of Task to be completed?
Initially, a user could enter into the nutrition plan the food they plan to eat. The following features have been added to the application:
  - [x] Adding an extra field for meals already eaten
  - [x] Fixing failing tests
  - [x] Adjusting the meal-plan table on the front end to display meal type column.

### How should this be manually tested?
Clone the repository to your local branch
```
$ git clone https://github.com/andela/wg-baqi.git
```
$ cd to the `wg-baqi` directory, create a virtual environment and install the dependencies
```
$ cd wg-baqi
$ virtualenv venv
$ source venv/bin/activate
$ pip install -r requirements.txt
```
Activate bootstrap, make migrations and run the server
```
$ bower install
$ invoke bootstrap-wger --settings-path ./wger/settings.py --no-start-server
$ python manage.py runserver
```
- Navigate to `Nutrition Plans` page

Screenshot 2019-06-20 at 08 47 13

- Click on `Add a new meal plan`. You can clearly see the option to add a Meal type as either `planned` or `consumed`

Screenshot 2019-06-20 at 08 48 33

- On adding the meal, the column for Meal Type can be clearly seen

Screenshot 2019-06-20 at 08 49 58

### CODE REVIEW

**Review by jmusila — COMMENTED** [review]
Tested locally, works as expected 👍

**Comment by wekesa931** [comment]
> Tested locally, works as expected 👍

👍🏽

File: wger/nutrition/forms.py [hunk 1]
```diff
@@ -128,6 +128,12 @@ class MealItemForm(forms.ModelForm):
         required=False)
     ingredient = forms.ModelChoiceField(queryset=Ingredient.objects.all(),
                                         widget=forms.HiddenInput)
+    Type_of_meal = (
+        ('Planned', 'Planned'),
```
Comment (wangonya) [inline wger/nutrition/forms.py:132]: Could you kindly clarify why the type being defined twice here?

**Review by wangonya — APPROVED** [review]
The feature is a nice addition to the nutrition functionality 👍🏾works well.

**Review by caveinn — APPROVED** [review]
Good work keep it up

**Pull request merged**